packages/gatsby-cli/bin.js

```diff
@@ -0,0 +1,3 @@
+// This file exists to fix a problem lerna has with symlinks on Windows
+// See https://github.com/gatsbyjs/gatsby/pull/2658
+require(`lib/index.js`)
```

Comment:
we probably need to move the shebang (`#!/usr/bin/env node`) from the index file to here as well